_articles/hu/finding-users.md

```diff
@@ -77,80 +77,80 @@ Használd ki a meglévő online közösségeket és platformokat, hogy elérd a
 
 Nézzük, hogyan lehet megtalálni a lényeges helyeket, ahol megoszthatod a projekted:
 
-* **Ismerd meg a kapcsolódó nyílt forráskódú projekteket és közösségeket.** Néha nem kell közvetlenül hirdetni a projekted. Ha a projekt tökéletes az adathalmaz kutatók számára, akik Python-t használnak, ismerkedj meg a Python adatkutató közösséggel. Amint az emberek megismerkednek, természetes lehetőség adódik arra, hogy beszéljenek és megoszthassák egymás munkáját.
-* **Találd meg azokat a problémákat, amelyet a projekted megold.** keresd meg azokat az embereket, akik érintettek lehetnek a megoldásban. Válaszold meg a kérdéseiket és találd meg azt a taktikát, amelynek segítségével elkezdik használni a projektedet.
-* **Kérj visszajelzést.** Mutatkozz be és mutasd be a projektedből azt a hallgatóságnak, amit érdekesnek vagy hasznosnak találnak. Legyél konkrét abban, hogy kinek hasznos a projekted. Próbáld így befejezni a mondatod: _"Úgy gondolom, hogy tényleg nagyon hasznos X csoportnak, akik az Y problémát akarják megoldani_". Hallgasd meg és válaszolj mások észrevételére, ne csak bemutató előadás legyen a projektedről.
+* **Ismerd meg a kapcsolódó nyílt forráskódú projekteket és közösségeket.** Néha nem kell közvetlenül hirdetni a projekted. Ha a projekt tökéletes a Python-t használó adathalmaz kutatók számára, ismerkedj meg a Python adatkutató közösséggel. Amint az emberek megismernek, természetes lehetőség adódik a beszélgetésre és arra, hogy megoszd velük a munkád eredményét.
+* **Találd meg azokat az embereket, akiknek a problémájára megoldást kínál a projekted.** Nézd át a kapcsolódó fórumokat, hogy megtaláld azokat az embereket, akik a projekted közönségét alkothatják. Válaszold meg a kérdéseiket és ha lehetséges, tapintatosan ajánld fel a projektedet megoldásként.
+* **Kérj visszajelzést.** Mutasd be magadat és amunkádat egy olyan közösségnek, amelyik érdekesnek találhatja azt. Legyél konkrét abban, hogy szerinted kinek hasznos a projekted. Próbáld így befejezni a mondatod: _"Úgy gondolom, hogy a projektem tényleg nagyon hasznos X csoportnak, akik az Y problémát akarják megoldani_". Hallgasd meg és válaszolj mások észrevételére, ne csak bemutató előadás legyen a projektedről.
```

Comment:
_és amunkádat_ helyett _és a munkádat_ kellene

Reply:
Úgy látom egy korrábbi PR-ben ezt javítottad, de most "visszatért" ez a hiba. A master ágba beolvasztottam a korábbi PR-t, ezért előtte a branche-edre pulloz, és ellenőrizd hogy javult-e. Köszi!

Reply:
igen, láttam, hogy olyan kommit is van a PR-ben, aminek nem kellene ott lenni. valószínűleg rebase-elnem kellett volna